modules/oe_content_sub_entity/tests/src/Kernel/SubEntityAccessControlHandlerTest.php

```diff
@@ -276,6 +276,24 @@ protected function accessTestScenarios(): array {
         'expected_result' => AccessResult::forbidden()->addCacheContexts(['user.permissions']),
         'status' => 0,
       ],
+      'user with access content / view label / published entity' => [
+        'permissions' => ['access content'],
+        'operation' => 'view label',
+        'expected_result' => AccessResult::allowed()->addCacheContexts(['user.permissions'])->addCacheTags(['node:2']),
+        'status' => 1,
+      ],
+      'user with access content / view label / unpublished entity' => [
+        'permissions' => ['access content'],
+        'operation' => 'view label',
+        'expected_result' => AccessResult::neutral(),
+        'status' => 0,
+      ],
+      'user with view unpublished sub entities / view label / unpublished entity' => [
+        'permissions' => ['view unpublished sub entities'],
+        'operation' => 'view label',
+        'expected_result' => AccessResult::forbidden()->addCacheContexts(['user.permissions']),
+        'status' => 0,
+      ],
```

Comment:
As I understand these test cases are applied for access `view label` operation of the **sub-entity**, but according to changes it has to be applied to **sub-entity type** (bundle).